packages/web-awesome/src/components/TestResult/TrInfo/index.tsx

```diff
@@ -71,14 +71,20 @@ export const TrInfo: FunctionalComponent<TrInfoProps> = ({ testResult }) => {
             <Loadable<Record<string, TestEnvGroup>, TestEnvGroup | undefined>
               source={testEnvGroupsStore}
               transformData={(groups) => groups[testResult.testCase.id]}
-              renderData={(group) => (
-                <TrTab id="environments" disabled={!group}>
-                  <div className={styles["test-result-tab"]}>
-                    {t("environments")}
-                    {Boolean(group) && <Counter size={"s"} count={Object.keys(group.testResultsByEnv).length} />}
-                  </div>
-                </TrTab>
-              )}
+              renderData={(group) => {
+                const envsCount = Object.keys(group?.testResultsByEnv ?? {}).length ?? 0;
+                const emptyEnvs = envsCount === 1 && DEFAULT_ENVIRONMENT in group.testResultsByEnv;
```

Comment:
The same logic is also used in `packages/web-awesome/src/components/TestResult/TrEnvironmentsView/index.tsx`. It could be extracted into a function that takes a `group` and returns the number of environments.